islands/opModal.tsx

```diff
@@ -36,10 +38,41 @@ export default function OpModal(
     grpcToken: string;
   },
 ) {
+  const displayType = opModal.value?.displayType;
+  const setDisplayName = () => {
```

Comment:
I think this should be named get not set as it's not setting anything. Also, a switch case might be a bit more readable than if/else